.github/workflows/ci.yml

```diff
@@ -61,49 +62,26 @@ jobs:
           git config --global user.email "ci@ci.invalid"
           git config --global user.name "Discourse CI"
 
-      - name: Setup packages
-        if: env.BUILD_TYPE != 'LINT'
-        run: |
-          sudo apt-get update
-          sudo apt-get -yqq install postgresql-client libpq-dev jpegoptim optipng jhead pngcrush pngquant
-          wget -qO- https://raw.githubusercontent.com/discourse/discourse_docker/master/image/base/install-pngquant | sudo sh
-
-      - name: Update imagemagick
-        if: env.BUILD_TYPE == 'BACKEND'
-        run: |
-          wget https://raw.githubusercontent.com/discourse/discourse_docker/master/image/base/install-imagemagick
-          chmod +x install-imagemagick
-          sudo ./install-imagemagick
-
       - name: Setup redis
         uses: shogo82148/actions-setup-redis@v1
```

Comment:
The image ships with redis already, this should be removed

Reply:
I know, theoretically the same with postgres.
But I leaving it as is for now intentionally - this setup allows us to easily experiment with different versions. Just pop another number in the version matrix and you're running tests on all declared versions.
Also, `Setup redis` step takes just 1-2s. 😃